### DISCUSSION TITLE
Why does opening the export or setting menus make the screen black with H265 hardware encoding enable? (not the playback of the video)

### DISCUSSION THREAD

**Discussion by YousufSSyed** [mifi/lossless-cut#1648] [Q&A]
I can see the H265 video fine and clip it in Losslesscut, but when I click on either settings or export, then the screen completely turns black. I looked at the FAQ and disabled H265 hardware decoding and it turns out fine. Why is that the case if the H265 video can play fine in the app for me?

**Comment by mifi** [comment]
probably something wrong with your graphics chipset or driver in your OS

**Reply by YousufSSyed** [reply]
I'm on an M1 MBP, do you have any ideas about it or how I could troubleshoot? I just checked and I have the ARM version of the app.

**Reply by mifi** [reply]
If possible you could try losslesscut with the same h265 video file on a different computer and see if it's the same problem. If it works on different computer, you could try to reinstall MacOS on your MBP, to see if it's an issue with your MacOS installation. Another thing you could try is to open losslesscut from the terminal and check if there's any error being logged when the screen turns black. Or check the Console app for system logs or crash logs at the time that the screen turns black. Or check the losslesscut developer tools (any error being printed in the javascript console when the screen turns black)

**Reply by YousufSSyed** [reply]
I turned H265 hardware decoding back on and I'm not having the bug again, even when I replace LosslessCut with a new copy (also the latest version).

Anyway, have you thought about investigating the cause of the issue?

**Reply by mifi** [reply]
I don't even know where to start. It's probably a bug in Chromium or MacOS, so I think not much I can do. The settings/export panel code is totally unrelated the video playback code.
